front/src/routes/ApplicantsPage.svelte

```diff
@@ -67,7 +82,7 @@
       .get()
       .json();
     rows = applications.map(application => ({
-      id: application.id,
+      id: parseInt(application.id),
```

Comment:
A bit nitpicking here, but `parseInt` should have a radix (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt)